### DISCUSSION TITLE
Issues with certain websites

### DISCUSSION THREAD

**Discussion by chastebetacuck** [silveredgold/beta-protection#64] [General]
On 4chan, pretty much every image returns an error.

On redditp.com and redditpx.com, the uncensored image shows before the placeholder kicks in.

On reddit.com, subreddit banners are also uncensored for a significant amount of time before the placeholder kicks in.

Latest version of chrome, Beta Protection v0.2.6, running Beta Censoring backend on MacOS Monterey 12.3. The issue is reproducible with the betasafety backend.

**Comment by chastebetacuck** [comment]
Issues reproduced in windows 11.

**Comment by silveredgold** [comment]
I don't use the Beta Safety backend so can't easily reproduce what you've described. Can you check if this still occurs with Beta Censoring, using v0.2.8 of Beta Protection?
